decorator.rb

```diff
@@ -0,0 +1,32 @@
+class Nemeable
```

Comment:
* Kindly fix this typo here `Nemeable` instead of `Nameable` causing errors as I try to test your class decoration. below is a screenshot of the error I get 
![image](https://user-images.githubusercontent.com/52497246/200536330-278388c5-4fc0-4dc5-b7ef-17ef0101d981.png)